tests/Candidate.test.js

```diff
@@ -0,0 +1,260 @@
+const Candidate = require('../Models/candidatemodel');
+const Job = require('../Models/jobsmodel');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const {
+    candidateSignup,
+    createToken,
+    checkAuthenticated,
+    checkNotAuthenticated,
+    comparePassword,
+    candidateLogout,
+    candidateLogin,
+} = require('../controllers/candidate');
+
+jest.mock('bcrypt', () => ({
+    hash: jest.fn().mockResolvedValue('hashed_password'),
+    compare: jest.fn().mockResolvedValue(true),
+}));
+jest.mock('jsonwebtoken', () => ({
+    sign: jest.fn().mockResolvedValue('sahfhasfhakshfkdhfa'),
+    verify: jest.fn(),
+}));
+
+
+describe('Candidate Controllers', () => {
+    describe('candidateSignup', () => {
+        it('it should perform the signup perfectly', async () => {
```

Comment:
You can remove `it`. With that test sentence will be like `candidateSignup should perform the signup perfectly` when you run the tests

Reply:
Resolved it!